force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -332,7 +448,7 @@ public with sharing class FlowControlLoopsCollections {
         // Get the salary of the employee
 
         // Return the salary of the employee, or null if the employee is not found in the Map
-        return null; // Replace null with the variable you used to store the result
+        return eemployeeSalaries.get(employeeName); // Replace null with the variable you used to store the result
```

Comment:
God work here. I would recommend being cautious that the input value was value. if the caller  passed in a key that was not in your Map you will get an error. 
You can check with the method containsKey() or you can use the null coalescing operator. 

 return salaryByEmployeeName.get(employeeName) ?? null;